Jellyfin.Api/Controllers/NotificationsController.cs

```diff
@@ -0,0 +1,137 @@
+        /// <param name="userManager">The user manager.</param>
+        public NotificationsController(INotificationManager notificationManager, IUserManager userManager)
+        {
+            _notificationManager = notificationManager;
+            _userManager = userManager;
+        }
+
+        /// <summary>
+        /// Endpoint for getting a user's notifications.
+        /// </summary>
+        /// <param name="userId">The user's ID.</param>
+        /// <param name="isRead">An optional filter by IsRead.</param>
+        /// <param name="startIndex">The optional index to start at. All notifications with a lower index will be dropped from the results.</param>
+        /// <param name="limit">An optional limit on the number of notifications returned.</param>
+        /// <returns>A read-only list of all of the user's notifications.</returns>
+        [HttpGet("{UserID}")]
+        public IReadOnlyList<NotificationDto> GetNotifications(
+            [FromRoute] string userId,
+            [FromQuery] bool? isRead,
+            [FromQuery] int? startIndex,
+            [FromQuery] int? limit)
+        {
+            return new List<NotificationDto>();
+        }
+
+        /// <summary>
+        /// Endpoint for getting a user's notification summary.
+        /// </summary>
+        /// <param name="userId">The user's ID.</param>
+        /// <returns>Notifications summary for the user.</returns>
+        [HttpGet("{UserID}/Summary")]
+        public NotificationsSummaryDto GetNotificationsSummary(
+            [FromRoute] string userId)
+        {
+            return new NotificationsSummaryDto();
+        }
+
+        /// <summary>
+        /// Endpoint for getting notification types.
+        /// </summary>
+        /// <returns>All notification types.</returns>
+        [HttpGet("Types")]
+        public IEnumerable<NotificationTypeInfo> GetNotificationTypes()
+        {
+            return _notificationManager.GetNotificationTypes();
+        }
+
+        /// <summary>
+        /// Endpoint for getting notification services.
+        /// </summary>
+        /// <returns>All notification services.</returns>
+        [HttpGet("Services")]
+        public IEnumerable<NameIdPair> GetNotificationServices()
+        {
+            return _notificationManager.GetNotificationServices();
+        }
+
+        /// <summary>
+        /// Endpoint to send a notification to all admins.
+        /// </summary>
+        /// <param name="name">The name of the notification.</param>
+        /// <param name="description">The description of the notification.</param>
+        /// <param name="url">The URL of the notification.</param>
+        /// <param name="level">The level of the notification.</param>
+        [HttpPost("Admin")]
+        public void CreateAdminNotification(
+            [FromForm] string name,
+            [FromForm] string description,
+            [FromForm] string? url,
+            [FromForm] NotificationLevel? level)
```

Comment:
The existing endpoint had these parameters as `ParameterType = "query"`

Reply:
Agree with your comment above about not changing the schema right now, so I'll change all these to be as they were, but I do think it would be good to have any POST/PUT requests have the data in the body of the request, since that's generally best practice. That should definitely be a separate PR though. Let me know your thoughts.

Reply:
I think it we should match the existing endpoints exactly for now, and then make improvements to them in a future PR like you said.

Changes to the API schema need to be carefully coordinate between all the client applications (both official and unofficial/private) so it's best to do that separately